Subject: ubuntu changes for smb.conf

+ Add "(Samba, Ubuntu)" to server string.
+ Comment out the default [homes] share, and add a comment about
 "valid users = %s" to show users how to restrict access to
  \\server\username to only username.
* Comment out "read only = yes" for a few standard shares.

diff --git a/debian/smb.conf b/debian/smb.conf
index 37fafb27398..385197abee9 100644
--- a/debian/smb.conf
+++ b/debian/smb.conf
@@ -30,2 +30,5 @@
 
+# server string is the equivalent of the NT Description field
+   server string = %h server (Samba, Ubuntu)
+
 #### Networking ####
@@ -168,5 +171,8 @@
 
-[homes]
-   comment = Home Directories
-   browseable = no
+# Un-comment the following (and tweak the other settings below to suit)
+# to enable the default home directory shares. This will share each
+# user's home directory as \\server\username
+;[homes]
+;   comment = Home Directories
+;   browseable = no
 
@@ -174,3 +180,3 @@
 # next parameter to 'no' if you want to be able to write to them.
-   read only = yes
+;   read only = yes
 
@@ -178,3 +184,3 @@
 # create files with group=rw permissions, set next parameter to 0775.
-   create mask = 0700
+;   create mask = 0700
 
@@ -182,3 +188,3 @@
 # create dirs. with group=rw permissions, set next parameter to 0775.
-   directory mask = 0700
+;   directory mask = 0700
 
@@ -186,6 +192,6 @@
 # with access to the samba server.
-# The following parameter makes sure that only "username" can connect
-# to \\server\username
+# Un-comment the following parameter to make sure that only "username"
+# can connect to \\server\username
 # This might need tweaking when using external authentication schemes
-   valid users = %S
+;   valid users = %S
 
